shell/src/main/java/alluxio/cli/fsadmin/command/GetFilePathCommand.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.fsadmin.command;
+
+import alluxio.annotation.PublicApi;
+import alluxio.cli.CommandUtils;
+import alluxio.cli.fsadmin.FileSystemAdminShellUtils;
+import alluxio.conf.AlluxioConfiguration;
+import alluxio.exception.status.InvalidArgumentException;
+import alluxio.wire.FileInfo;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+
+import java.io.IOException;
+
+/**
+ * Command for getting file path from a file id.
+ */
+@PublicApi
+public class GetFilePathCommand extends AbstractFsAdminCommand {
+  private static final String HELP_OPTION_NAME = "h";
+  private static final Option HELP_OPTION =
+      Option.builder(HELP_OPTION_NAME)
+          .required(false)
+          .hasArg(false)
+          .desc("print help information.")
+          .build();
+
+  private final AlluxioConfiguration mConf;
+
+  /**
+   * @param context fsadmin command context
+   * @param alluxioConf Alluxio configuration
+   */
+  public GetFilePathCommand(Context context, AlluxioConfiguration alluxioConf) {
+    super(context);
+    mConf = alluxioConf;
+  }
+
+  @Override
+  public String getCommandName() {
+    return "getFilePath";
+  }
+
+  @Override
+  public int run(CommandLine cl) throws IOException {
+    if (cl.hasOption(HELP_OPTION_NAME)) {
+      System.out.println(getUsage());
+      System.out.println(getDescription());
+      return 0;
+    }
+
+    FileSystemAdminShellUtils.checkMasterClientService(mConf);
+
+    long fileId;
+    String arg = cl.getArgs()[0];
+    try {
+      fileId = Long.parseLong(arg);
+    } catch (NumberFormatException e) {
+      throw new InvalidArgumentException(arg + " is not a valid file id.");
+    }
+
+    try {
+      FileInfo fileInfo = mFsClient.getFilePath(fileId);
```

Comment:
We can achieve the same thing with two steps: 

1. get the file path from file id with `client.getFilePath`; 
2. get detailed file info by calling `client.getStatus` with the path acquired from step 1.

Step 2 is the same thing as `alluxio fs stat /file`. It should be optional and should hide behind a `--stat` flag, which suggests it does an extra step to fetch the file stats as reported by `fs stat`.

Reply:
On second thought, I think we should totally make `alluxio fs stat` accept a file id, and add a new format option to output the file path.